examples/default/domain/aggregate.go

```diff
@@ -75,7 +75,7 @@ func (i *InvitationAggregate) HandleCommand(command eh.Command) error {
 			return nil
 		}
 
-		i.StoreEvent(&InviteAccepted{i.AggregateID()})
+		i.StoreEvent(&InviteAccepted{i.AggregateID().(eh.UUID)})
```

Comment:
Maybe all domain objects can use the `ID` interface to avoid this potentially unsafe type assert?